src/testing/time_refinement_study.cpp

```diff
@@ -116,6 +116,18 @@ int TimeRefinementStudy<dim, nstate>::run_test() const
         const double dt =  params.ode_solver_param.initial_time_step;
         const int n_timesteps= flow_solver->ode_solver->current_iteration;
         pcout << " at dt = " << dt << std::endl;
+
+
+        if (this->all_parameters->ode_solver_param.runge_kutta_method == PHiLiP::Parameters::ODESolverParam::RK3_2_5F_3SStarPlus 
+            && this->all_parameters->ode_solver_param.atol == 1e-4 && this->all_parameters->ode_solver_param.rtol == 1e-4 
+            && this->all_parameters->time_refinement_study_param.number_of_times_to_solve == 1){
+            double L2_error_expected = 2.14808703658e-5; 
+            pcout << " Expected L2 error is: " << L2_error_expected << std::endl;
+            if (L2_error > L2_error_expected + 1e-16 || L2_error < L2_error_expected - 1e-16){
```

Comment:
I would recommend using a slightly larger tolerance (1E-15 maybe) - the tighter tolerance might result in the test failing just due to very small implementation differences which cause rounding to act differently. This is a great test to ensure that the LSRK is working well though!